.clang-tidy

```diff
@@ -3,19 +3,29 @@
 InheritParentConfig: true
 Checks: '
 bugprone-*,
+-bugprone-argument-comment,
```

Comment:
This one is a reasonable check too?

Reply:
We can enable it but then many comments have to be changed.